src/components/sessions/data.js

```diff
@@ -56,8 +58,8 @@ module.exports = [
 
   {
     isBreak: true,
-    duration: 30,
-    title: 'Cake break (30 mins)',
+    duration: 25,
+    title: 'Cake break (25 mins)',
```

Comment:
Can't tell from the diff, but is this the afternoon break? If so, let's change `cake` to `ice cream`.

Reply:
Yes this is the afternoon break. I'll change it to "ice cream" then 👍

Reply:
Fixed